lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -9,15 +9,33 @@ def patch_renderer
     end
 
     def patch_renderer_render_template
-      if defined?(::ActionView::Renderer)
-        ::ActionView::Renderer.class_eval do
+      if defined?(::ActionView::TemplateRenderer)
+        ::ActionView::TemplateRenderer.class_eval do
+          alias_method :render_without_datadog, :render
           alias_method :render_template_without_datadog, :render_template
-          def render_template(*args, &block)
-            ActiveSupport::Notifications.instrument('start_render_template.action_view')
-            render_template_without_datadog(*args, &block)
+
+          def render(context, options)
+            # create a tracing context and start the rendering span
+            @tracing_context = {}
+            ::ActiveSupport::Notifications.instrument('start_render_template.action_view', tracing_context: @tracing_context)
+            render_without_datadog(context, options)
+          rescue StandardError => e
```

Comment:
Since you re-raise the error here, you might consider rescuing `Exception` (the idea is that Exception all the StandardError cases plus cases where... you need to re-raise the exception, such as a signal, an out-of-memory, etc. but generally speaking, if you re-raise -> it's fine to use Exception, and can give more insight in edge cases)

Reply:
right! thanks!